log-sink/src/test/java/org/springframework/cloud/stream/module/log/LogSinkApplicationTests.java

```diff
@@ -39,11 +52,39 @@
 	@Bindings(LogSink.class)
 	private Sink sink;
 
+	@SuppressWarnings("unused")
 	@Autowired
 	private Sink same;
 
+	@Autowired
+	private LoggingHandler logSinkHandler;
+
+	@BeforeClass
+	public static void setUp() {
```

Comment:
I was following the existing model in the FTP source.

If that model is incorrect, it should be fixed before asking "newbies" to contribute :smile: